libraries/rush-lib/src/logic/operations/PhasedOperationPlugin.ts

```diff
@@ -29,13 +29,18 @@ function createOperations(
   existingOperations: Set<Operation>,
   context: ICreateOperationsContext
 ): Set<Operation> {
-  const { projectsInUnknownState: changedProjects, phaseSelection, projectSelection } = context;
+  const {
+    projectsInUnknownState: changedProjects,
+    phaseOriginal,
```

Comment:
`watchPhases` and `initialPhases` are not required to have any overlap whatsoever, so in the watch mode branch `phaseOriginal` should probably be set to `watchPhases`.

Reply:
Do you mean this?
https://github.com/microsoft/rushstack/pull/3754/commits/575f8f0d076c1c6d862166f4a672f51ced6bcb7a

Reply:
Yes. That would work.

Reply:
Thanks